tests/legacy-cli/e2e/tests/commands/builder-not-found.ts

```diff
@@ -14,7 +14,11 @@ export default async function () {
       /Could not find the '@angular-devkit\/build-angular:browser' builder's node package\./,
     );
     await expectToFail(() =>
-      execAndWaitForOutputToMatch('ng', ['build'], /Node packages may not be installed\./),
+      execAndWaitForOutputToMatch(
+        'ng',
+        ['build'],
+        /Node packages may not be installed\. Try installing with 'npm install'\./,
```

Comment:
The package manager command will change if the test is run in a project using a different package manager (e.g., yarn).
`getActivePackageManager` can be used to get the name of the package manager used in the test.

Reply:
Yeah, I figured CI would fail with this but wasn't sure how to get the current package manager. Updated.